package.json

```diff
@@ -8,7 +8,7 @@
     "build-dev": "webpack --watch",
     "start-dev": "nodemon ./server/index.js",
     "build": "webpack",
-    "start": "node ./server/index.js",
+    "start": "NODE_ENV=production node ./server/index.js",
```

Comment:
This may not be needed, because in production, we are using docker secrets not .env